.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+Tag name (required for release branches):
+Originator(s):
+
```

Comment:
Do we want a `"Description"` section which is what the user thinks might be the overall summary of the PR when they open it.  Maybe the title captures this, but I'm not sure.  I'm trying to think about the two stages for this description, what is known when the PR is opened and what it is at the very end, right before merge.

Reply:
@cacraigucar @nusbaume how do we feel about changing "Summary" to "Description"? I think maybe "Summary" is what the PR title is and "Description" opens the door for more detail?

Reply:
I am happy with changing `Summary` to `Description` in the template.

Reply:
Sounds good to me

Reply:
change made!